rfc9114.md

```diff
@@ -301,11 +302,11 @@ host identifier to an IP address, establishing a QUIC connection to that address
 on the indicated port (including validation of the server certificate as
 described above), and sending an HTTP/3 request message targeting the URI
 to the server over that secured connection.  Unless some other mechanism is used
-to select HTTP/3, the token "h3" is used in the Application-Layer Protocol
+to select HTTP/3, the token "h3" is used in the Application Layer Protocol
```

Comment:
I disagree with this one, RFC 7301 only uses the hyphenated term of art "Application-Layer Protocol Negotiation Extension"

Reply:
```suggestion
to select HTTP/3, the token "h3" is used in the Application-Layer Protocol
```
Agreed; RFC 7301 wins.